yunwoo/programmers-1/부족한-금액-계산하기.js

```diff
@@ -0,0 +1,14 @@
+function solution(price, money, count) {
+  let result = 0;
+  for (i = 1; i <= count; i++) {
+    result += price * i;
+  }
+  return result < money ? 0 : result - money;
+}
```

Comment:
```suggestion
function solution(price, money, count) {
  let result = 0;
  for (i = 1; i <= count; i++) {
    result += price * i;
  }
  return result < money ? 0 : result - money;
}
```
- 군더더기 없이 깔끔하네요 고생하셨습니다 🙆‍♂️ 
- 제 풀이 solution2보시면 배열 메서드로만 해서 푼게있는데 Array.from 문법하고 한번 참고해주세요 🙏

Reply:
확인했습니다 참고하겠습니다👍